internal/internal_workflow_testsuite.go

```diff
@@ -178,6 +181,7 @@ type (
 		testError        error
 		doneChannel      chan struct{}
 		workerOptions    WorkerOptions
+		dataConverter    DataConverter
```

Comment:
This also comes from `workerOptions`.
I initially added this to the shared struct above and it was the bug I was chasing this morning.